### PR TITLE
GH 821: links on pptx

### PR DESCRIPTION
**Pull request by ejulio-ts**
Follow up of https://github.com/terrastruct/d2/pull/1139

This adds links to `.pptx` files.

testing on PowerPoint and KeyNote on Mac

https://user-images.githubusercontent.com/102628724/230991930-f2b7aed3-e0c9-4015-bf8b-7788dc013815.mp4

### CODE REVIEW

**Review by alixander — CHANGES_REQUESTED** [review]
is there a way to show mouse cursor for external links?

other than these small things, looks good to go

**Comment by gavin-ts** [comment]
as another issue, I wonder if the appendix should take in a root name so `1: root > mobile` matches the slide title `out / mobile` based on filename `out`?

**Comment by ejulio-ts** [comment]
> as another issue, I wonder if the appendix should take in a root name so 1: root > mobile matches the slide title out / mobile based on filename out?

That's something I was thinking too. Another alternative is to not name the board based on the output file and use `root`

**Comment by ejulio-ts** [comment]
@alixander what do you mean here

> is there a way to show mouse cursor for external links?

Keep the standard cursor instead of pointer for external links?

**Comment by alixander** [comment]
Screen Shot 2023-04-11 at 7 11 19 AM

no i want all links to be cursor pointer. here, the mouse doesn't switch to cursor pointer?

**Comment by ejulio-ts** [comment]
It seems to be a bug in PowerPoint. When I click on a slide link and then move the mouse over another link, it loses the pointer. I need to click again to get it back.

https://user-images.githubusercontent.com/102628724/231221021-0aa359a8-5c97-48a7-8737-cb53a229676e.mp4

File: lib/png/png.go [hunk 1]
```diff
@@ -19,6 +19,9 @@ import (
 	"oss.terrastruct.com/util-go/xmain"
 )
 
+// ConvertSVG scales the image by 2x
+const SCALE = 2.
```
Comment (alixander) [inline lib/png/png.go:23]: use it in `png` too

**Review by alixander — APPROVED** [review]

**Pull request merged**